### ISSUE TITLE
Remove cleaning/migrations steps from installer

### ISSUE THREAD

**Issue by jooola** [libretime/libretime#1372]
Now that the migration path from airtime is clear and require a fresh system anyway, we should focus on having a clean installer without all the migrations tasks.

Libretime is currently still in Alpha, and this allow us to do breaking changes without worrying too much.

I also propose a dedicated script for those who want to cleanup an old airtime/libretime install to start fresh on the same system.
Basically if you don't start with a fresh system, you will have to clean the systems with this dedicated script.

In addition, possible breaking changes between airtime/libretime alpha versions can be handled by adding a dedicated entry with the required steps in the change-log, and the possibility to use the cleanup script for the lazy ones.

**Comment by Robbt** [comment]
This sounds like a good idea but I also suspect will take some time to rewrite the install script and pull out the migration/upgrade steps.

**Comment by jooola** [comment]
> This sounds like a good idea but I also suspect will take some time to rewrite the install script and pull out the migration/upgrade steps.

I don't mind, if we think it is the right thing to do, we should do it.

I also made a rewrite of the installer here #1352
